baseline.yaml

```
@@ -649,6 +649,38 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+  - id: OSPS-DO-18   
+    maturity_level: 2
+    category: Documentation
+    criteria: |
+       The project MUST perform threat modeling and
+       attack surface analysis to understand and protect
+       against critical code paths, functions, and interactions 
+       with the system.
+    objective: |
+       Projects need to conduct threat modeling, attack 
+       surface analysis, and data-flow analysis in order
```

```suggestion
       Projects need to conduct threat modeling and attack 
       surface analysis in order
```

Just say threat modeling. Data flow analysis typically follows on as part of threat modeling, but you can do lots of data flow analysis that has nothing to do with security, and there's no need to require it. Just require the threat modeling and I think you're covered.